python/lsst/ip/isr/isrFunctions.py

```diff
@@ -134,6 +134,27 @@ def makeThresholdMask(maskedImage, threshold, growFootprints=1, maskName='SAT'):
     return measAlg.Defects.fromFootprintList(fpList)
 
 
+def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
+    """Grow a mask by an amount and add to the BAD plane.
```

Comment:
This docstring isn't correct after the behavior change. BAD is just your default value.